x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -252,6 +269,10 @@ export interface ActionDetails {
   startedAt: string;
   /** The date when the action was completed (a response by the endpoint (not fleet) was received) */
   completedAt: string | undefined;
+  /**
+   * The output data from an action
+   */
+  outputs?: Record<string, ActionResponseOutput<TOutputContent>>;
```

Comment:
Can we please rename this to `output`. 🙏

Reply:
This is a collection, so it does make sense to have it in plural (collection of outputs per agentId)

Reply:
In that case, why don't we format it as a list? Easier to process it on the UI.

`outputs?: Array<{ agentId: string } & ActionResponseOutput>;`

Reply:
We talked offline and this is something we discussed during office hours last week that I missed. 😅 . So all is good here.